src/pages/onBoarding/index.tsx

```diff
@@ -2,20 +2,59 @@ import {
   ContainerBoardModal,
   ContainerOnBoarding,
   OnBoardImage,
+  ModalContainer,
+  TabsContainer,
+  Tab,
+  TabLabel,
+  TabWrapper,
+  TabLine,
 } from '@/styles/pages/onBoarding'
 import onBoardImage from '@/assets/onBoarding/Ilustrações.svg'
-import OnBoardModal from '@/components/molecules/OnBoardModal'
-import { withAuth } from '@/context/Auth/withAuth'
+import GridSpecialities from '@/components/atoms/GridSpecialities'
+import PerfilTab from '@/components/molecules/PerfilTab'
+import { useState } from 'react'
+import { OnBoardingProvider } from '@/context/OnBoardingContext'
+import { EditPhotoProvider } from '@/context/EditPhotoContext'
+
+export default function OnBoarding() {
+  const [step, setStep] = useState<1 | 2>(1)
```

Comment:
A forma como você fez está totalmente correta, mas fica uma sugestão:

Imaginando que futuramente podemos ter mais steps, é sempre bom já tentarmos manter as tipagens externalizadas para fácil manutenção, sugestão:

```ts
type StepNumber = 1 | 2;

export default function OnBoarding() {
  const [step, setStep] = useState<StepNumber>(1)
```

Dessa forma, sempre que precisar adicionar novos steps, você modifica apenas na interface, sem precisar poluir muito o useState.

Reply:
Essa tipagem `1 | 2` é usada em outras interfaces e estados. Eu deveria criar um arquivo separado pra ela, ou exportá-la diretamente de `pages/onBoarding` ou talvez de `context/onBoardingContext`?

Reply:
> Essa tipagem `1 | 2` é usada em outras interfaces e estados. Eu deveria criar um arquivo separado pra ela, ou exportá-la diretamente de `pages/onBoarding` ou talvez de `context/onBoardingContext`?

Se é algo que quando mudar, vai mudar para todos os demais componentes, você poderia colocar a tipagem e exportar diretamente do `context/onBoardingContext`, agora se for algo especifico daquela página, poderia apenas colocar no começo do arquivo sem exportação.